simple-stack/src/main/java/com/zhuinden/simplestack/ScopeManager.java

```diff
@@ -432,6 +507,10 @@ private boolean canFindFromScopeAll(String scopeTag, String serviceTag) {
             }
         }
 
+        if(globalServices.hasService(serviceTag)) {
+            return globalServices.getService(serviceTag);
```

Comment:
This should have a different order if the global is considered an explicit rather than an implicit parent of all scope chains.

Reply:
nah